Gemfile

```diff
@@ -33,6 +33,7 @@ gem 'sprockets', '~> 3'
 gem 'sprockets-rails', '>= 2.3.3', '< 3'
 gem 'responders', '~> 2.0'
 gem 'roadie-rails', '~> 1.1'
+gem 'email_validator', '1.6.0'
```

Comment:
For the record, we already have an email validator based on regexp in the codebase. Maybe to reuse it? Grep for `validates :mail`.